UGW: use different patch directory if PATCH_SUFFIX is defined in target.mk (added for GRX500 support) - temperory patch only

diff --git a/include/kernel.mk b/include/kernel.mk
--- a/include/kernel.mk
+++ b/include/kernel.mk
@@ -30,7 +30,7 @@ else
   endif
 
   ifeq ($(TARGET_BUILD),1)
-    PATCH_DIR ?= ./patches$(if $(wildcard ./patches-$(KERNEL_PATCHVER)),-$(KERNEL_PATCHVER))
+    PATCH_DIR ?= ./patches$(if $(wildcard ./patches-$(KERNEL_PATCHVER)),-$(KERNEL_PATCHVER))$(if $(PATCH_SUFFIX),-$(PATCH_SUFFIX))
     FILES_DIR ?= $(foreach dir,$(wildcard ./files ./files-$(KERNEL_PATCHVER)),"$(dir)")
   endif
   KERNEL_BUILD_DIR ?= $(BUILD_DIR_BASE)/linux-$(BOARD)$(if $(SUBTARGET),_$(SUBTARGET))$(if $(BUILD_SUFFIX),_$(BUILD_SUFFIX))
